### PR TITLE
blackoilfoammodules: put parameters in separate class

### PR DESCRIPTION
**Pull request by akva2**
limits the amount of static variable declarations

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: opm/models/blackoil/blackoilfoammodules.hh [hunk 1]
```diff
@@ -29,26 +29,25 @@
...
 #include "blackoilproperties.hh"
-//#include <opm/models/io/vtkblackoilfoammodule.hh>
-#include <opm/models/common/quantitycallbacks.hh>
 
-#include <opm/material/common/Tabulated1DFunction.hpp>
-//#include <opm/material/common/IntervalTabulated2DFunction.hpp>
+#include <opm/common/OpmLog/OpmLog.hpp>
+
+#include <opm/models/blackoil/blackoilfoamparams.hh>
+//#include <opm/models/io/vtkblackoilfoammodule.hh>
```
Comment (blattms) [inline opm/models/blackoil/blackoilfoammodules.hh:36]: why keep this and the other one not?

**Review by blattms — APPROVED** [review]
Just a question, otherwise approved.

I am getting a bit dizzy from all the parameters.

File: opm/models/blackoil/blackoilfoammodules.hh (hunk 1 above)
Comment (akva2) [reply opm/models/blackoil/blackoilfoammodules.hh:36]: i guess  ican remove it. my reasoning was that the first one is not a todo, while the second kinda is.

**Comment by akva2** [comment]
jenkins build this please

**Pull request merged**